Cargo.toml

```diff
@@ -37,69 +37,68 @@ sovd-interfaces = { path = "cda-sovd-interfaces" }
 
 # ---- common crates ----
 async-trait = "0.1.89"
-thiserror = "2.0.16"
-hashbrown = "0.15"
+thiserror = "2.0.17"
+hashbrown = "0.16"
 num-traits = "0.2"
-parking_lot = "0.12.4"
+parking_lot = "0.12.5"
 serde = "1.0"
 serde_json = "1.0"
-tokio = { version = "1.43.0", default-features = false }
+tokio = { version = "1.48.0", default-features = false }
 futures = "0.3"
-regex = "1.11.1"
-uuid = "1.16.0"
+regex = "1.12.2"
+uuid = "1.18.1"
 prost = "0.14"
 flatbuffers = "25.9.23"
-xz2 = "0.1.6"
+xz2 = "0.1.7"
 hex = "0.4.3"
 strum = "0.27.2"
 strum_macros = "0.27.2"
-indexmap = "2.9.0"
+indexmap = "2.12.0"
 http = "1.3.1"
 mime = "0.3.17"
-jsonwebtoken = "9.3.1"
-chrono = "0.4.40"
+jsonwebtoken = { version = "10.2.0", features = ["rust_crypto"] }
```

Comment:
todo: jsonwebtoken now marks `insecure_disable_signature_validation` as deprecated. it should be replaced with `jsonwebtoken::dangerous::insecure_decode` before merging this.